MIGRATION_GUIDE.md

```diff
@@ -9,20 +9,60 @@ across different versions.
 
 ## v0.97.0 ➞ v0.98.0
 
-### *(new feature)* connection resource
-Added a new resource for managing connections.
+### _(new feature)_ connection resources
```

Comment:
no clue either, i used pre-push and additionally linter on its own to check whether all is as it should. I will change it back to `*` but i have no clue what it has been changed. Maybe because of different formatting tool (?)

Reply:
still, please validate such changes before the re-review